### ISSUE TITLE
Les erreurs de récupération des informations sur un export en cours ne sont pas toutes correctement affichées pour l'utilisateur

### ISSUE THREAD

**Issue by lucaa** [DISIC/observatoire#1077]
Voir la description d'un tel cas d'erreur possible ici: https://github.com/DISIC/observatoire/issues/1076#issuecomment-1030578984 .

Le problème que ce ticket doit traiter est que, si la situation d'erreur ci-dessus arrive, l'affichage de la barre de progrès ne change pas par rapport à une attente normale d'informations du serveur. Ainsi, l'utilisateur ne sait pas qu'il y a eu un problème de récupération de données et que l'export n'est pas bloqué, il a l'impression que c'est 'en attente'. Il devrait y avoir une information qu'il y a eu une erreur de récupération des infos (et donner la possibilité de re-essayer ou donner plus d'informations pour dépannage).

Un cas d'utilisation à vérifier est que se passe-t-il si le poste utilisateur est déconnecté de l'internet pendant l'export?

**Comment by lucaa** [comment]
> [5308879.pdf](https://github.com/DISIC/observatoire/files/8255920/5308879.pdf) Pour info, rapport pingdom de l’Observatoire du 13 février au 15 mars

@AnthonyBrunelli je ne suis pas sure comment ces informations sont relevantes dans le cadre de ce ticket, peut-être que le problème a été compris différemment?

**Comment by AnthonyBrunelli** [comment]
Non, c'est juste "à toute fin utile" qu'on t'a transmis le document, tu es juge de sa pertinence ou non :)
